### PR TITLE
Add Tania to steering council

### PR DESCRIPTION
**Pull request by jaimergp**
Dear @conda/steering-council,

The voting period for @trallard's application has concluded:

image

The vote reached quorum (10/14) and has **passed** (7/7).

This PR adds Tania to the CSV. @trallard, please review the changes to include your email address and pronouns. Thanks!

### CODE REVIEW

File: steering.csv [hunk 1]
```diff
@@ -14,3 +14,4 @@ jaimergp,jaime.rogue@gmail.com,Jaime Rodríguez-Guerra,Quansight,he/him
 kkraus14,keith.j.kraus@gmail.com,Keith Kraus,no funding, he/him
 baszalmstra,zalmstra.bas@gmail.com,Bas Zalmstra,prefix.dev,he/him
 beckermr,becker.mr@gmail.com,Matthew R. Becker,no funding,he/him
+trallard,,Tania Allard,Quansight,
```
Comment (trallard) [inline steering.csv:17]:
```suggestion
trallard,taniar.allard@gmail.com,Tania Allard,Quansight,she/her
```

**Comment by jaimergp** [comment]
I think you can approve the PR now @trallard :)

**Review by trallard — APPROVED** [review]

**Comment by jaimergp** [comment]
Welcome aboard!

**Comment by wolfv** [comment]
Welcome!

**Comment by ocefpaf** [comment]
Nice to have you in conda-forge Tania!

**Comment by jezdez** [comment]
Welcome Tania!

**Pull request merged**